script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -73,76 +73,76 @@ The vampire's fighting spirit has not faded.
 ...No.
 To be more precise, it's like he is actually ready to fight for the first time.
 <Page21>
-"I see. Then accept your good fortune, Vlov. Normally, I would have finished you before you woke up. The one that saved your life was a despicable murderer. Well, for a short time at least."
+"I see. Then accept your good fortune, Vlov. Normally, I would have finished you before you woke up. But your life was saved by a despicable murderer. Well, for a short time at least."
 <Page22>
 "―――Indeed.
-The one in front of me is far different from the one I have heard about in the stories.
+You're quite different in person from what I had been led to expect.//伝え聞いた貴様の姿と、今の貴様はかけ離れている。
 ...It appears that in this moment of truth, fortune smiles upon me."
 <Page23>
-"I hate to say it, but that's right.
-It seems that both of us only have one final attack remaining. In the end, one of us will commence the attack and the other will come to an end.
-But before that, allow me to send some final gratitude."
+"I hate to say it, but you're right.
+It seems that both of us only have one final attack remaining. Whichever of us attacks, the other will disappear.
+But before that, allow me to extend to you at least this courtesy."
 <Page24>
-"―――Answer me, puny livestock of mine own servitude.
-Testimony of Ancestor. Lesion of the Root. Hollow like a grand vortex.
+"―――Answer me, puny livestock of mine own servitude.//Runt? mine own servitude reads like Arc is the thrall
+Testimony of the Ancestor. Lesion of the Root. Hollow like a grand vortex.//祖の<証|あかし>。<根|ね>の病巣。大渦の如きその穴蔵。
 Descendant carrying new Principle.
 What doth blood mean to thee?"
 <Page25>
-It was a stern voice, far removed from the current situation.
+It was a stern voice, jarringly out of place in the current situation.
 Arcueid's question could be taken as a reprimand.
-After opening his eyes wide in surprise, Vlov says the following.
+After opening his eyes wide in surprise, Vlov replies.
 <Page26>
 "―――Allow me to respond with respect.
 For me, blood is the very essence of warmth.
-The heat of life. The elixir to cure my frostbite. That which turns my world and my Principle.
+The heat of life. The elixir to cure my frostbite. Around it revolves my world, and my Principle.//Blood's what makes his world go round, phrasing I chose here is a bit less literal but I think translates better to both subjects. 我が世界、我が原理を回すもの。
 Indeed. Without blood, I would freeze to death in this blistering cold."
-The man reveals his own true colors in a voice that could be described as respectful.
+The man reveals his true colors in a voice that could be described as respectful.
 <Page27>
 "...I remember now.
-A freak who hated and feared humans, drifting across an immense sea and setting up a castle to avoid them.
+A freak who hated and feared humans, who drifted across an immense sea and built a castle to avoid them.//人間を嫌い、恐れ、避けようと絶海を漂流し、城を構えた変わりもの。
```

Comment:
OK with "who drifted _across_ distant seas"? If this is all in service of avoiding humans, it's by crossing a sea you avoid them, rather than by bobbing around on it